openedx/features/partners/models.py

```diff
@@ -33,3 +33,11 @@ def __unicode__(self):
     class Meta:
         unique_together = ('user', 'partner')
 
+
+class PartnerCommunity(models.Model):
+    category_id = models.CharField(max_length=255)
```

Comment:
for the sake of consistency throughtout the platform (edx side apart from nodebb connection apps) can we rename this to community_id?

Reply:
Done